deps/install_tensorflow_1_15.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eou pipefail
+
+pip install tensorflow==1.15.0
+
+# Find out where do the tensorflow libs and sources reside
+TF_DIR="$(python -c 'import tensorflow as tf; print(tf.sysconfig.get_lib())')"
+
+# Make TF lib detectable for the ldd linker
+ln -s "$TF_DIR/libtensorflow_framework.so.1" "$TF_DIR/libtensorflow_framework.so"
+
+# Work around TF issue: https://github.com/tensorflow/tensorflow/issues/22766
+sed -i 's/return ABSL_ASSERT(len <= kMaxSize), len;/return len;/' "$TF_DIR/include/absl/strings/string_view.h"
```

Comment:
As far as I remember, this was only needed for GPU? In that case, we should not do that  for Travis.

Reply:
(#87 says this is for nvcc only...)

Reply:
OK I'll try to remove the Bash script altogether.